### ISSUE TITLE
Unable to run "go get"

### ISSUE THREAD

**Issue by jpf** [okta/okta-jwt-verifier-golang#61]
I'm running Go 1.15.1:

```
$ go version
go version go1.15.1 darwin/amd64
```
If I have no Okta packages installed: 

```
$ find ~/go -type d -name okta
$
```
(Note that I'm using the default $GOPATH of `~/go`)

Then when I run `go get -u github.com/okta/okta-jwt-verifier-golang` I get an error (2):

```
$ go get -u github.com/okta/okta-jwt-verifier-golang
go: github.com/lestrrat-go/option upgrade => v1.0.0
go: github.com/lestrrat-go/jwx upgrade => v1.1.1
go: github.com/lestrrat-go/iter upgrade => v1.0.0
go: github.com/goccy/go-json upgrade => v0.4.4
go: golang.org/x/crypto upgrade => v0.0.0-20201221181555-eec23a3978ad
# github.com/okta/okta-jwt-verifier-golang/adaptors/lestrratGoJwx
../../go/pkg/mod/github.com/okta/okta-jwt-verifier-golang@v1.0.0/adaptors/lestrratGoJwx/lestrratGoJwx.go:40:17: undefined: jwk.FetchHTTP
../../go/pkg/mod/github.com/okta/okta-jwt-verifier-golang@v1.0.0/adaptors/lestrratGoJwx/lestrratGoJwx.go:70:16: undefined: jws.VerifyWithJWKSet
$ echo $?
2
```

What am I doing wrong?

**Comment by laura-rodriguez** [comment]
Hi @jpf !

@bogdanprodan-okta @bretterer any ideas?

**Comment by bogdanprodan-okta** [comment]
@jpf https://github.com/okta/okta-jwt-verifier-golang/pull/62 this should fix the issue

**Comment by jpf** [comment]
Thanks @bogdanprodan-okta! I can see  that you made the change, but I'm still having trouble on my machine:

```
$ find ~/go -type d -name okta
$ go get -u github.com/okta/okta-jwt-verifier-golang
go: downloading github.com/okta/okta-jwt-verifier-golang v1.0.0
go: github.com/lestrrat-go/option upgrade => v1.0.0
go: github.com/lestrrat-go/jwx upgrade => v1.1.1
go: golang.org/x/crypto upgrade => v0.0.0-20201221181555-eec23a3978ad
go: github.com/goccy/go-json upgrade => v0.4.4
go: github.com/lestrrat-go/iter upgrade => v1.0.0
# github.com/okta/okta-jwt-verifier-golang/adaptors/lestrratGoJwx
../../go/pkg/mod/github.com/okta/okta-jwt-verifier-golang@v1.0.0/adaptors/lestrratGoJwx/lestrratGoJwx.go:40:17: undefined: jwk.FetchHTTP
../../go/pkg/mod/github.com/okta/okta-jwt-verifier-golang@v1.0.0/adaptors/lestrratGoJwx/lestrratGoJwx.go:70:16: undefined: jws.VerifyWithJWKSet
```

**Comment by jpf** [comment]
Ah, I see the change has been merged into the develop branch ... and it looks like y'all are merging it in to the main branch now. I'll keep an eye on the status and try again once your #62 is merged.

**Comment by bretterer** [comment]
@jpf we just tagged v1.1.0.  Can you please try updating your application and see if your error is resolved?

**Comment by jpf** [comment]
It works! Thanks so much!
